mkosi/__init__.py

```diff
@@ -2493,8 +2493,34 @@ def invoke_apt(
         INITRD="No",
     )
 
-    run_workspace_command(args, root, cmdline, network=True, env=env)
+    # Ensure apt inside the image can see the local mirror outside of it
+    if args.offline_build and args.mirror and args.mirror.startswith("file:"):
+        nspawn_params = [f"--bind-ro={args.mirror[5:]}"]
+    else:
+        nspawn_params = None
```

Comment:
Wouldn't bind mounting the repo on `args.mirror` starting with `file:` make sense generally irrespective of `args.offline`?

Reply:
yeah I guess so